docs/docs/use-cases/day-2-operations.md

```diff
@@ -43,14 +43,15 @@ completed for the version set in the labels/annotations.
 
 To illustrate the update of a [workload](https://kubernetes.io/docs/concepts/workloads/),
 let's assume the following example, including
-a [workload](https://kubernetes.io/docs/concepts/workloads/) called `podtato-head-frontend` that includes a pre-task and
-a pre-evaluation.
+a [workload](https://kubernetes.io/docs/concepts/workloads/) called `podtato-head-frontend` that includes a
```

Comment:
I don't think it's necessary to make every instance of "workload" a link.  It seems like having a link for the first use of the term within the section should be enough.  That is what our current guidelines say but it is certainly open to discussion.

Reply:
agreed - I removed all links except the first one now